hack/scripts/dev-update-image-references.py

```diff
@@ -12,7 +12,7 @@ def getLatestManifest():
     pipelineDir = os.path.join(os.getcwd(), "bin/pipeline")
     if os.path.exists(pipelineDir):
         shutil.rmtree(pipelineDir)
-    repo = Repo.clone_from("https://github.com/stolostron/backplane-pipeline.git", pipelineDir)
+    repo = Repo.clone_from("ssh://git@github.com/stolostron/backplane-pipeline.git", pipelineDir)
```

Comment:
Was this changed accidentally?  It doesn't look related to the pr.

Reply:
nope, this was changed on purpose, though it isn't related to the technical work. The https git clone requires a username/password in plaintext, which isn't supported by github anymore, but ssh is, so I made this change